docs/en/guides/containers/oss/eks/self-managed-to-managed-oss.md

```diff
@@ -0,0 +1,204 @@
+# EKS Observability : Managed Observability Services to AWS Managed Open Source Observability Services
+In the ever-evolving world of modern software development, observability has become a critical aspect of ensuring the reliability, performance, and scalability of applications. While self-managed tools like Prometheus and Datadog have been invaluable in providing insights into the health and behavior of systems, the increasing complexity and scale of modern architectures often demand more robust and scalable solutions. Enter managed observability services, such as Amazon Managed Prometheus and Amazon Managed Grafana. These fully managed offerings from AWS aim to simplify the management and operation of observability tools, freeing up valuable time and resources for organizations to focus on their core business objectives.
+
+In this guide, we will explore the journey of migrating from self-managed observability tools to Amazon's managed services.
```

Comment:
We are talking about self-managed solutions and 3rd party solutions in this article.

Reply:
Pushed an update